packages/ember-cli-fastboot/test/request-details-test.js

```diff
@@ -11,8 +12,8 @@ function injectMiddlewareAddon(app) {
     pkg.devDependencies['body-parser'] =
       process.env.npm_package_devDependencies_body_parser;
     pkg.dependencies = pkg.dependencies || {};
-    pkg.dependencies['fastboot-express-middleware'] =
-      process.env.npm_package_dependencies_fastboot_express_middleware;
+    pkg.dependencies['fastboot'] = `file:${path.resolve(__dirname, '../../fastboot')}`
+    pkg.dependencies['fastboot-express-middleware'] = `file:${path.resolve(__dirname, '../../fastboot-express-middleware')}`
```

Comment:
This fixes `Legacy Mocha Tests`, where `fetch` was `undefined` during initialisation.